src/components/dev-hub/feedback/feedback-container.js

```diff
@@ -39,12 +39,12 @@ const modalStyles = css`
     min-width: ${MODAL_WIDTH};
 
     @media ${screenSize.upToMedium} {
+        border: none;
         height: 100%;
-        min-width: unset;
         max-width: unset;
-        width: 100%;
-        border-radius: 0;
```

Comment:
<img width="399" alt="Screenshot 2021-05-17 at 22 53 46" src="https://user-images.githubusercontent.com/79590294/118548249-bec21300-b762-11eb-8516-4af3a6315978.png">
Let's keep border-radius for fixing the border at the bottom of the modal.